gis/bylaw_text_to_centreline/README.md

```diff
@@ -0,0 +1,300 @@
+THEN (
+    CASE WHEN ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom)
+    > ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom_translated)
+    THEN ST_Union(_wip2.pgrout_centreline ,
+    ST_LineSubstring(_wip2.whole_centreline, 
+    ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom) - (metres_btwn1/ST_Length(ST_Transform(_wip2.whole_centreline, 2952))),
+    ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom) )  
+        )
+ 
+    WHEN ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom)
+    < ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom_translated)
+    THEN ST_Union(_wip2.pgrout_centreline ,
+    ST_LineSubstring(_wip2.whole_centreline, 
+    ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom),
+    ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom) + (metres_btwn1/ST_Length(ST_Transform(_wip2.whole_centreline, 2952)))  )
+        )
+    
+    END )
+
+--routed centreline - part of centreline xx metres from that intersection that got trimmed
+WHEN ST_Within(ST_Transform(ST_ClosestPoint(_wip2.whole_centreline, _wip2.oid1_geom_translated), 2952), ST_BUFFER(ST_Transform(_wip2.pgrout_centreline, 2952), 2, 'endcap=square join=round')) = TRUE
+THEN (
+    CASE WHEN ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom)
+    < ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom_translated)
+    THEN ST_Difference(_wip2.pgrout_centreline ,
+    ST_LineSubstring(_wip2.pgrout_centreline, 
+    ST_LineLocatePoint(_wip2.pgrout_centreline, _wip2.oid1_geom),
+    ST_LineLocatePoint(_wip2.pgrout_centreline, _wip2.oid1_geom) + (metres_btwn1/ST_Length(ST_Transform(_wip2.pgrout_centreline, 2952)))  )
+        )
+
+    WHEN ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom)
+    > ST_LineLocatePoint(_wip2.whole_centreline, _wip2.oid1_geom_translated)
+    THEN ST_Difference(_wip2.pgrout_centreline ,
+    ST_LineSubstring(_wip2.pgrout_centreline, 
+    ST_LineLocatePoint(_wip2.pgrout_centreline, _wip2.oid1_geom) - (metres_btwn1/ST_Length(ST_Transform(_wip2.pgrout_centreline, 2952))),
+    ST_LineLocatePoint(_wip2.pgrout_centreline, _wip2.oid1_geom) )  
+        )
+
+    END )
+
+ELSE _wip2.pgrout_centreline
+
+END);
+```
+
+The image below shows the four possible cases (A1, A2, B1 and B2)
+![image](https://user-images.githubusercontent.com/54872846/77707649-284df380-6f9c-11ea-9509-b0db0cdf24c5.png)
+
+
+## Confidence output
+
+The function outputs a confidence level (`con`), which is defined after the `centreline_segments` variable. This value represents how close the names of the input values are to the intersections they were matched to. Specifically, this value is the sum of the levenshtien distance between the `highway2` value and the street name it matched to and the `btwn1`/`btwn2` value and the street name it matched to. It is the sum of this value for both intersections.
+
+# Quality Control
```

Comment:
This section should list out how to identify these problems in the output. Is it because of `WARNING` messages thrown by the function? Or are some of these identified from manual inspection, but now we now why (e.g. "street name is duplicated across former municipalities")

It should also be split into issues identified and a new "Outstanding work" section referencing the open tasks that could solve some quality control problems.